### ISSUE TITLE
exportIndividualSimulations creates simulations in inconsistent state

### ISSUE THREAD

**Issue by Yuri05** [Open-Systems-Pharmacology/OSPSuite-R#398]
Even though some parameters are changed in the simulation: the corresponding building block status is green.
Example (inputs: s. #397): result of `exportIndividualSimulations `after reloading it in MoBi

PSV:
![grafik](https://user-images.githubusercontent.com/25061876/83198607-d06c6e00-a13f-11ea-8200-c954c2805315.png)

Simulation:
![grafik](https://user-images.githubusercontent.com/25061876/83198652-e843f200-a13f-11ea-91c6-be9557bdf1cb.png)

Values of `Organiim|p1` etc. were overwritten in the simulation; but the status of all building blocks is "unchanged":

![grafik](https://user-images.githubusercontent.com/25061876/83198727-145f7300-a140-11ea-9499-3e4c9683efc9.png)

**Comment by msevestre** [comment]
Ospsuite core does not update building blocks in any shape or form. this can't be fixed

**Comment by Yuri05** [comment]
That's fine, but I wonder why the used PSV is shown as "unchanged" in the simulation

**Comment by msevestre** [comment]
because there is no concept of building block in R. We always assume consistent state in a pkml simulation. Setting a parameter value `parameter.value = 5` will not update the PSV of the simulation. This is something that MoBi does with commands etc.. 

In our case, it's almost as if we had NO building blocks so to speak. They are in the PKML, but are not used anywhere. We could think about adding entries for each change to the PSV, but this would require a serious rewrite as we would not be allowed to access parameter directly

Nothing we can do here at the moment

**Comment by msevestre** [comment]
I really wonder how we can make this work. R basically bypasses everything and changes the simulation directly.

**Comment by Yuri05** [comment]
Maybe it should happen when saving a simulation from R to pkml?
R would not call pkml export directly. Instead it would call something like "SynchronizeAndSaveToPkml", where parameter values are written to the PSV before exporting to pkml.

**Comment by msevestre** [comment]
like all parameter values?

**Comment by Yuri05** [comment]
no, only those with `IsFixedValue = True`

**Comment by msevestre** [comment]
I don't think this is set for constant parameters. Only for formula. May be wrong

**Comment by Yuri05** [comment]
Is set also for constant.
https://github.com/Open-Systems-Pharmacology/OSPSuite.Core/blob/1bc3551091c210944c7b635624e4c9d52fff614a/src/OSPSuite.Core/Domain/Quantity.cs#L104-L129
